cli/repair/clusterwide_config.go

```diff
@@ -0,0 +1,314 @@
+package repair
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	configDirName        = "config"
+	topologyConfFilename = "topology.yml"
+
+	keyInstances            = "servers"
+	keyReplicasets          = "replicasets"
+	keyInstanceAdvertiseURI = "uri"
+	keyInstanceDisabled     = "disabled"
```

Comment:
Why is there no `keyInstanceExpelled` or `disabledState `?

Reply:
There is no `expelled` key, in case of expelled instance it's config value is `expelled`.
So, `disabled` is an instance config key, `expelled` is a whole instance config value.

E.g.:

```yaml
servers:
  srv-1:
    advertise_uri: localhost:3303
    disabled: true
  drv-2: expelled
```